external-tests/sriov/test.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
what is this file for?

Reply:
The file is required here:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/hack/run-functests.sh#L41
For now I don't think we need it to perform anything.

Reply:
It's ok not to have it. That file is a way to run tests from other sources than the local testsuite.
It's used for example for running the performance operator's tests directly from its repo.

Reply:
afair it exists at line 60 if the file is not present, so either we have it or we have to modify run-functest.sh

Reply:
At line 60 it exits if at least one test among the suite and the external ones fails.

Reply:
Tried it out, works without the script indeed.
Removing.